### ISSUE TITLE
Prevent user from signing out while background process is occuring

### ISSUE THREAD

**Issue by tierninho** [github/VisualStudio#1145]
VS crashed while the git clone window was opening and loading, as I chose to click `signout` to see what would happen. Perhaps we should grey the`signout` link when another process is occurring or move it to a location more obscure.

Test build: 2.3.0.16/GitHub.VisualStudio.vsix

<img width="1118" alt="screen shot 2017-08-16 at 10 48 35 am" src="https://user-images.githubusercontent.com/14828183/29369514-cf16f26e-8270-11e7-8fae-760fce2b9fd0.png">

**Comment by grokys** [comment]
Hmm the clone dialog should be a modal dialog, so this shouldn't be possible. Is the dialog not showing modally for you?

**Comment by tierninho** [comment]
Yes, the git clone was a modal dialogue (the white space hidden behind the crash dialogue).

**Comment by grokys** [comment]
How did you click "sign out" if the modal dialog was showing? The whole of VS apart from the dialog should be disabled.

**Comment by tierninho** [comment]
Yes it is, but there appears to be a lapse. You have to be quick and click the `sign out` immediately after clicking clone.

As this is an extreme edge case, feel free to close it.

**Comment by grokys** [comment]
Moved this to low priority because you have to really _try_ to get this to happen!
